libraries/exporters/src/main/java/org/eclipse/sw360/exporter/ReleaseHelper.java

```diff
@@ -41,14 +79,11 @@ public int getColumns() {
 
     @Override
     public List<String> getHeaders() {
-        return HEADERS;
+        return addAdditionalData() ? HEADERS_EXTENDED_BY_ADDITIONAL_DATA : HEADERS;
     }
 
     @Override
     public SubTable makeRows(Release release) throws SW360Exception {
-        if (!release.isSetAttachments()) {
-            release.setAttachments(Collections.emptySet());
```

Comment:
`release` is key of the map `releaseToShortenedStringsMap` which is why it should not be modified here. Otherwise the keys won't work anymore cause the hashCode changes after the object has been changed and the value assigned to the key cannot be identified from now on.